otter/convergence/gathering.py

```diff
@@ -196,25 +197,31 @@ def update_drained_at(node):
     yield do_return(list(filter(bool, nodes)))
 
 
+_DRAINING_CREATED_RE = (
```

Comment:
> wonder if it would be appropriate to open a ticket on CLBs to get this information in a more structured way, sans regexes (if you haven't already).

It would be appropriate. However, I really doubt they would even look at it considering the fact that CLB is EOLing (for past 2 years 😉). Moreover, this is an unintentional API that according to them no one uses :). I think I'll create a ticket anyway just so that they are aware about it.